contracts/plugins/trading/DutchTradeRouter.sol

```diff
@@ -0,0 +1,111 @@
+// SPDX-License-Identifier: BlueOak-1.0.0
+pragma solidity 0.8.19;
+
+import { SafeERC20 } from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import { IDutchTradeCallee, TradeStatus, DutchTrade } from "../trading/DutchTrade.sol";
+import { IMain } from "../../interfaces/IMain.sol";
+
+contract DutchTradeRouter is IDutchTradeCallee {
```

Comment:
Let's document the class and give a brief description of how to use the contract as a bidder here